automata/pda/pda.py

```diff
@@ -24,6 +40,280 @@ class PDA(Automaton, metaclass=abc.ABCMeta):
     initial_stack_symbol: str
     acceptance_mode: PDAAcceptanceModeT
 
+    @staticmethod
+    def _get_state_name(state_data: PDAStateT) -> str:
```

Comment:
Isn't this function identical to the other `_get_state_name` function? It would be great if that logic could be shared somehow, even if it means putting this function into a utility file or something.